### ISSUE TITLE
Organization proposal

### ISSUE THREAD

**Issue by danisev7** [Encode-Solidity-Bootcamp-Group-10/Week2#3]
To make the pull requests for this project, I propose to do a lightweight feature branches.

It will be the same feature branches strategy proposed at [Gitflow Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) but without the use of development branch because this is a very small project.

![image](https://user-images.githubusercontent.com/1956176/178116761-8cbcf68c-6e84-49c5-8d59-11685e2a5d92.png)

Basically the workflow will be:

To find a peace of code to improve
To open a new branch with the name feature-xxxxx
To do a pull request for this branch

**Comment by Super-Nim** [comment]
Ok, so a new branch for each specific feature sounds good.

But we need to encourage people to review/approve the changes first. Otherwise it'll sit there forever or just end up being merged without review

**Comment by davidpablosg** [comment]
The only problem I can forecast with this approach is that some conflict resolution knowledge on Git is required.
Aside from it, it is the best way to work in parallel.
